docs/bsp.md

```diff
@@ -147,6 +147,10 @@ trait BuildTarget {
     * The id.uri is used if None. */
   def displayName: Option[String]
 
+  /** The type of build target. Useful for an IDE to show
+    * targets with different kinds in the UI. */
+  def kind: Int
```

Comment:
`App` is missing. What about projects that are for example of kind bench AND test?  I think it would be better to have `labels: List[String]` and leave it to clients and servers to come up with meaningful names.